src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,165 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exeptions.ValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService service;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        service = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+    }
+
+    @Test
+    void register_nullUser_notOk() {
+        assertThrows(ValidationException.class, () ->
+                service.register(null));
+    }
+
+    @Test
+    void register_sameLogin_notOk() {
+        user.setLogin("Max1999");
+        user.setPassword("123456");
+        Storage.people.add(user);
+        User newUser = new User();
+        newUser.setLogin("Max1999");
+        assertThrows(ValidationException.class, () ->
+                service.register(newUser));
+    }
+
+    @Test
+    void register_nullLogin_notOk() {
+        user.setLogin(null);
+        assertThrows(ValidationException.class, () ->
+                service.register(user));
+    }
+
+    @Test
+    void register_lessThanSixLoginLength_notOk() {
```

Comment:
```suggestion
    void register_tooShort_notOk() {
```
if you decide to change limit to 8 later, you may forget to change test method name